Agents/PubChemAgent/pubchemagent/pug/chebi_classification.py

```diff
@@ -0,0 +1,90 @@
+from bioservices import ChEBI
+from pubchemagent.kgoperations.querykg import kg_operations
+from pubchemagent.kgoperations.addkgdata import create_uuid
+from pubchemagent.kgoperations.getkgdata import get_uuid
+from pubchemagent.kgoperations.querytemplates import *
+from pubchemagent.utils.default_configs import UPDATE_ENDPOINT
+import logging
+
+logging.getLogger('urllib3').setLevel(logging.CRITICAL)
+logging.getLogger('requests').setLevel(logging.CRITICAL)
+logging.getLogger('requests_cache').setLevel(logging.CRITICAL)
+logging.getLogger('bioservices').setLevel(logging.CRITICAL)
+logging.getLogger('matplotlib').setLevel(logging.CRITICAL)
+logging.getLogger('suds').setLevel(logging.CRITICAL)
```

Comment:
A lot of the files seem to be starting with this kind of block. I wonder if this is a good idea?

Reply:
I added a comment to explain that this block is necessary to supress excessive log outputs